spynnaker/pyNN/models/spike_source/spike_source_poisson_vertex.py

```diff
@@ -356,8 +331,6 @@ def rate(self, rate):
 
     @property
     def start(self):
-        if self.__is_variable_rate:
```

Comment:
Why remove these error messages?

Reply:
coz the dsg calls this now. regalrdd of if its variable rate. it ust to call its private variable, but not possible when at machine level. instead of having a seperate function, felt it was better to just not bother doing a check. it still has a value if its runnign in flex mode afterall